package.json

```diff
@@ -21,6 +21,7 @@
 		"express": "*",
 		"grunt": "*",
 		"canvas": "*",
+		"basic-request": "*",
```

Comment:
We should unify our "requests" libs (we have request and basic-request). I am happy to use "basic-request" if it's simpler.